<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Filippo</author>
 <class>StationEditDialog</class>
 <widget class="QDialog" name="StationEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="stationTab">
      <attribute name="title">
       <string>Station</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="generalBox">
         <property name="title">
          <string>General</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="stationNameEdit">
            <property name="placeholderText">
             <string>Station name...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Short name:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="shortNameEdit">
            <property name="placeholderText">
             <string>Abbreviation...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Type:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="stationTypeCombo">
            <item>
             <property name="text">
              <string>Normal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Simple Stop</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Junction</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Phone:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="phoneEdit">
            <property name="placeholderText">
             <string>Phone number...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="svgGroupBox">
         <property name="title">
          <string>SVG Image</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Add Image</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Remove Image</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Save to file</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>Show</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gatesTab">
      <attribute name="title">
       <string>Gates</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="addGateButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="removeGateButton">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="gatesView">
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>2</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Default In Track</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Out track count</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>A</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item row="0" column="3">
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>B</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="1" column="2">
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item row="1" column="3">
          <property name="text">
           <string>1</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tracksTab">
      <attribute name="title">
       <string>Tracks</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QToolButton" name="addTrackButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="removeTrackButton">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="moveTrackUpBut">
           <property name="text">
            <string>Move Up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="moveTrackDownBut">
           <property name="text">
            <string>Move Down</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="trackView">
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>2</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>3</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Color</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Length</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Platform Length</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Freight Length</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Max Axes</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Electrified</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Trough</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>1</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>Orange</string>
          </property>
          <property name="background">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>255</red>
             <green>170</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string>80 cm</string>
          </property>
          <property name="textAlignment">
           <set>AlignTrailing|AlignVCenter</set>
          </property>
         </item>
         <item row="0" column="3">
          <property name="text">
           <string>70 cm</string>
          </property>
          <property name="textAlignment">
           <set>AlignTrailing|AlignVCenter</set>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
         </item>
         <item row="0" column="4">
          <property name="text">
           <string>75 cm</string>
          </property>
          <property name="textAlignment">
           <set>AlignTrailing|AlignVCenter</set>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
         </item>
         <item row="0" column="5">
          <property name="text">
           <string>120</string>
          </property>
          <property name="textAlignment">
           <set>AlignTrailing|AlignVCenter</set>
          </property>
         </item>
         <item row="0" column="6">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
         </item>
         <item row="0" column="7">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>2</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="1" column="2">
          <property name="text">
           <string>60 cm</string>
          </property>
          <property name="textAlignment">
           <set>AlignTrailing|AlignVCenter</set>
          </property>
         </item>
         <item row="1" column="4">
          <property name="text">
           <string>58 cm</string>
          </property>
          <property name="textAlignment">
           <set>AlignTrailing|AlignVCenter</set>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
         </item>
         <item row="1" column="5">
          <property name="text">
           <string>80</string>
          </property>
          <property name="textAlignment">
           <set>AlignTrailing|AlignVCenter</set>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>3</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="2" column="2">
          <property name="text">
           <string>50 cm</string>
          </property>
          <property name="textAlignment">
           <set>AlignTrailing|AlignVCenter</set>
          </property>
         </item>
         <item row="2" column="4">
          <property name="text">
           <string>48 cm</string>
          </property>
          <property name="textAlignment">
           <set>AlignTrailing|AlignVCenter</set>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
         </item>
         <item row="2" column="5">
          <property name="text">
           <string>70</string>
          </property>
          <property name="textAlignment">
           <set>AlignTrailing|AlignVCenter</set>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="trackConnectionsTab">
      <attribute name="title">
       <string>Track Connections</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QToolButton" name="addTrackConnBut">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="removeTrackConnBut">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="trackConnView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gateConnections">
      <attribute name="title">
       <string>Gate Connections</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QToolButton" name="addGateConnBut">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="removeGateConnBut">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="editGateConnBut">
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="gateConnView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StationEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StationEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
